core/src/server/rpc_module.rs

```diff
@@ -738,6 +740,90 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 	}
 }
 
+/// Represent a pending subscription which waits being accepted or rejected.
+///
+/// Warning: you need to call either `PendingSubscription::accept` or `PendingSubscription::reject` otherwise
+/// the subscription will not make any progress.
```

Comment:
```suggestion
/// the subscription will be dropped with an `InvalidParams` error.
```

Or something like that?